### DISCUSSION TITLE
revalidate:600 - in reality every few seconds

### DISCUSSION THREAD

**Discussion by orrkislev** [vercel/next.js#17243] [Help]
Hey

On our landing page (with a lot of traffic!) getStaticProps makes a few calls to MongoDB Atlas so it takes a few seconds to complete and it is quite heavy on the server.
we don't make a lot of changes to the page content, but it is very handy to have incremental static regeneration, as rebuilding and deploying the whole project just to update a few small changes in the DB is annoying, and takes a long time (more than 1500 pages), so I set it up with reavalidate:600. (also tried other numbers)
But then I see in the server logs that the calls are being made every few seconds, sometimes more than once a second. this take up huge bandwidth on my server and I had to remove the ISR, sadly..

I guess it has something to do with the amount of traffic, but I really dont know..

(this also happen on other pages)
btw: nextJS 9.5.3, deployed on vercel

thanks!

**Comment by howells** [comment]
I'm seeing this too: I have revalidation set from 300 - 3600 seconds, yet my server is being hit incredibly hard, and I'm struggling to understand what is going on...

Have you found out more?

**Reply by valse** [reply]
I think this is related to this issue #17758

**Comment by valse** [comment]
I'm facing a strange behavior with the revalidate timing too (I have a next.js app deployed in a single nodejs x64 process on a Windows server with a lot of pages and requests too).

If I set a revalidate time of 180 (seconds) the page (a blog post) get updated after a more long time or it never updated too... if I set a revalidate time of 1 (second) the page get updated after 1 or few seconds later... I can't understand how the regeneration process work :/

Thanks
